src/translations/en/common.json

```diff
@@ -83,7 +83,7 @@
   "openPortSelection": "Open Port Selection",
   "versionUnsupported": "{{name}} version '{{version}}' (Layout {{layout}}) is not yet supported.<br/><br/>Let us know by opening an <a href=\"https://github.com/stylesuxx/esc-configurator\" title=\"https://www.github.com/issues/new\" target=\"_blank\">issue</a>.",
   "motorControl": "Motor Control",
-  "motorControlText": "<p>Make sure your ESCs are properly set up to reflect the state of the sliders.</p><p>Eg.: When you have 3D mode enabled, make sure the ESCs are also set up for 3D mode, otherwise the motors might go off with full power.</p><p>Also be aware that the motors will not spin if you have bi-directional Dshot enabled on the Flight-controller, but not on the ESC. Which might be the case when flashing from RPM emabled firmware to BlHeli_S.</p>",
+  "motorControlText": "<p>Make sure your ESCs are properly set up to reflect the state of the sliders.</p><p>Eg.: When you enabled 3D mode in your FC, make sure the ESCs are also set up for 3D mode, otherwise the motors might go off with full power.</p><p>Also be aware that the motors will not spin if you have bi-directional Dshot enabled on the Flight-controller, but ESC does not support it. Which might be the case when flashing from RPM emabled firmware to BlHeli_S.</p>",
```

Comment:
Shouldn't it rather be:
> When you enabled 3D mode on your flight controller,...
and
> ...but the ESC does not support it.

Reply:
ahh, this was edit online so, yeah, my bad.

"flight controller" looks relatively long compare to "FC", as almost every user in our app have already accept "FC = flight controller" so I think this should be fine. If you can provide further meaningful reason(s) I will change.

and yes, a typo and an error that I miss "the" in front of "ESC", will fix

Reply:
I used "flight controller' in other places, so I would just like to be consistent.

Reply:
oh, I didn't notice that. Consistency is also important.